spec/controllers/notifications_controller_spec.rb

```diff
@@ -56,8 +56,15 @@
     end
 
     it 'succeeds for notification dropdown' do
+      Timecop.travel(6.seconds.ago) do
+        @notification.touch
+      end
       get :index, :format => :json
       expect(response).to be_success
+      note_html = JSON.parse(response.body)[0]['also_commented']['note_html']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.